Shoko.Server/Databases/MySQL.cs

```diff
@@ -720,6 +720,7 @@ public class MySQL : BaseDatabase<MySqlConnection>, IDatabase
         new(110, 1, "CREATE TABLE `AniDB_FileUpdate` ( `AniDB_FileUpdateID` INT NOT NULL AUTO_INCREMENT, `FileSize` BIGINT NOT NULL, `Hash` varchar(50) NOT NULL, `HasResponse` BIT NOT NULL, `UpdatedAt` datetime NOT NULL, PRIMARY KEY (`AniDB_FileUpdateID`) );"),
         new(110, 2, "ALTER TABLE `AniDB_FileUpdate` ADD INDEX `IX_AniDB_FileUpdate` (`FileSize` ASC, `Hash` ASC) ;"),
         new(110, 3, DatabaseFixes.MigrateAniDB_FileUpdates),
+        new(111, 1, "Update AniDB_Episode SET DateTimeUpdated = '1970-01-01 00:00:00';"),
```

Comment:
Why are we resetting this time for all the episodes?

Reply:
Yes, I could have worded it better.

Reply:
The migration can be removed if you or @da3dsoul think it should be removed. (That's why I made it a separate commit, in case someone else deemed it unnecessary.)

Reply:
I said this before, but why use an individual date for episodes? We don't get a date from AniDB, and the date will only be updated when the anime is. Just use the AniDB_AnimeUpdate date

Reply:
And I said it before too, we do get individual dates for episodes for when they were last updated. They are stored in the xml files and I mapped them to be used in this PR.

Also, said date can be used to check which of the episode entries — comparing the record stored in the database and the entry in the xml file (which can be loaded from a stall/older cache if banned) — is the freshest, and conditionally update the episode records (e.g. set the overview, rating, length, etc., move the episode to another anime, etc.) if the xml file contains a newer representation than the one stored in the database.

Reply:
I forgot

Reply:
So, should we remove the database migration? I will reset the dates on my own install either way, but I thought it would be more helpful for others if they was just reset as part of a migration.

Reply:
> So, should we remove the database migration?

Still waiting for an answer. If nobody responds then is it safe to assume the answer is "no, it's fine to let it be" and I can just merge this now?

Reply:
I would make a helper function that pulls it from the xml